src/Device.Net/ControlTransferResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Device.Net
+{
+#pragma warning disable CA1815 // Override equals and operator equals on value types
+    public readonly struct ControlTransferResult
```

Comment:
@josesimoes let's rename ReadResult to TranserResult ,and rename BytesRead to BytesTransferred. Then we can delete this class.

You can do that in your branch, or I can change that after the PR. Let me know.

Reply:
Suggest that we tackle that after this is merged, so the PR is more atomic.